### ISSUE TITLE
Update Prerequisites from React-Native to ReactJS

### ISSUE THREAD

**Issue by HeroicHitesh** [anitab-org/anitab-org.github.io#255]
### Describe the bug
Update Prerequisites from React-Native to ReactJS

### To Reproduce
1. Go to 'https://github.com/anitab-org/anitab-org.github.io#prerequisites'
2. See Prerequisites

### Expected behavior
Prerequisites should have ReactJS in place of React-Native

### Additional context
@annabauza 's [Comment](https://github.com/anitab-org/anitab-org.github.io/issues/42#issuecomment-780130118) on GitHub
[Conversation](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio/topic/Questions/near/229421330) regarding this issue on Zulip

IMO we can also update the tags of the repo to reflect this change and start migrating the code one component at a time.

**Comment by nandini45** [comment]
@shades-7  it is a first timer issue 
available for the new comers only

**Comment by anjali481** [comment]
Can I work on this issue?

**Comment by nandini45** [comment]
@anjali481 yeah . here i assigned it to you. make sure to submit the work update after 3 days. 😊

**Comment by anjali481** [comment]
@nandini45  please review the PR

**Comment by nandini45** [comment]
this issue has been merged
